components/popconfirm/__tests__/index.test.tsx

```diff
@@ -274,7 +274,7 @@ describe('Popconfirm', () => {
     fireEvent.click(triggerNode);
     expect(onOpenChange).toHaveBeenLastCalledWith(true, undefined);
     fireEvent.keyDown(triggerNode, { key: 'Escape', keyCode: 27 });
-    expect(onOpenChange).toHaveBeenLastCalledWith(false, eventObject);
+    expect(onOpenChange).toHaveBeenLastCalledWith(false, undefined);
```

Comment:
Why this test case changed?

Reply:
After the click event, eventObject is undefined. Then, the 'Escape' key is pressed to quit. Why is eventObject expected in this case?

Also, I'm not familiar with testing. I think that checking only the first state change from true to false in onChange.mock.calls should be enough

Reply:
if expect, what should i change, i'm not familiar with testing

Reply:
i don't whether if because this
https://github.com/ant-design/ant-design/blob/b3728baf3cedab63647e9c6604f0055f787f7ec0/components/popconfirm/index.tsx#L87-L93

`settingOpen` second argument is `?`, it seems onOpenChange.mork.call [boolean,undefined] is becasue of this

https://github.com/ant-design/ant-design/blob/b3728baf3cedab63647e9c6604f0055f787f7ec0/components/popconfirm/index.tsx#L61-L67

Reply:
i try to pass test case because i thought it was unnecessary
now i know the problem